### PR TITLE
Plans: update Jetpack plans table with emphasis on security and performance

### PR DESCRIPTION
**Pull request by keoshi**
### Changes proposed in this Pull Request

* Plans: update Jetpack plans table, emphasis on security and performance.
* Equivalent changeset on Jetpack: D25583-code

### Testing instructions

* Fire up this PR.
* Visit the following links and make sure you see the updated table:
  * `/plans/SITE_URL`
  * `/plans/monthly/SITE_URL`

### Before

image

### After

image

#### Breakdown of all the changes:

- **Personal plan**:
  - Remove ~Off-site~ from `Daily Backups`.
  - Add `Expanded Site Activity` entry on Personal plan, before Priority Support.
    - Description:
    - > A comprehensive and filterable list of all the activity happening on your site takes the guesswork out of site management, debugging, and repair.
- **Premium plan**:
  - Add `Automatic Security Fixes` entry on Premium plan, before Unlimited Video Hosting.
    - Description:
    - > Automated and immediate resolution for a large percentage of known security vulnerabilities or threats.
  - `Unlimited Video Hosting` moves up before Site Monetization.
  - Remove `Social Media Automation`.
- **Professional plan**:
  - Update plan description:
    - > The most powerful WordPress sites: real-time backups, enhanced search, and unlimited premium themes.
  - ~Off-site~ `Real-time Backups` moves to the top.
  - `Enhanced Site-wide Search` moves to second place right after Real-time Off-site Backups.
- **Global**:
  - Embolden certain aspects of features.

Partially addresses #31356

### CODE REVIEW

File: client/lib/plans/constants.js [hunk 1]
```diff
@@ -1081,13 +1083,11 @@ export const PLANS_LIST = {
...
-				'The most powerful WordPress sites: unlimited premium ' +
-					'themes, real-time backups, and enhanced search.'
+				'The most powerful WordPress sites: real-time backups, ' +
+					'enhanced search, and unlimited premium themes.'
 			),
 		getTagline: () =>
-			i18n.translate(
-				'You have full access to premium themes, marketing tools, and priority support.'
-			),
+			i18n.translate( 'You have full the full suite of security and performance tools.' ),
```
Comment (torres126) [inline client/lib/plans/constants.js]: Typo here btw. :) 

```suggestion
			i18n.translate( 'You have the full suite of security and performance tools.' ),
```

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -1019,13 +1023,11 @@ export const PLANS_LIST = {
...
```
Comment (torres126) [inline client/lib/plans/constants.js]: Typo here btw. :) 

```suggestion
			i18n.translate( 'You have the full suite of security and performance tools.' ),
```

File: client/lib/plans/constants.js (hunk 1 above)
Comment (keoshi) [reply client/lib/plans/constants.js]: Thanks for noticing that! Fixed in https://github.com/Automattic/wp-calypso/pull/31465/commits/124f5d37220233d46c0395acbbe0e33475a52a64

File: client/lib/plans/constants.js (hunk 2 above)
Comment (keoshi) [reply client/lib/plans/constants.js]: Thanks again! 😸

**Pull request merged**